pkg/resource/logging-credentials/logging_operator_secrets.go

```diff
@@ -36,6 +37,17 @@ func LoggingCredentialsSecretMeta() metav1.ObjectMeta {
 	return metadata
 }
 
+func TracingCredentialsSecretMeta() metav1.ObjectMeta {
```

Comment:
I think we do not need this for loki tbh. The secrets are in the ingress secret already right? Can we not read them from there directly?

Reply:
Oh no we do because ran authmap is in htpasswrod format rigth?